bin/dfx-sns-demo-mksns-parallel

```diff
@@ -69,14 +70,14 @@ become_user() {
 create_all_sns() {
   PROCESSES=()
   echo "I am $$"
-  for ((i = 0; i < NUM_SNS_TO_MAKE; i++)); do
+  for ((i = 1; i <= NUM_SNS_TO_MAKE; i++)); do
```

Comment:
Yes, it starts with 0 because when creating a snapshot, the first SNS was created by another script here - https://github.com/dfinity/snsdemo/pull/370/files#diff-275424577f68bbaf8ba7ae9c4c8d8384792f24b0566fd26fae6fe904af2e30fbR23

Reply:
This script shouldn't assume that it's being called after another script by yet another script.
Can you add a `--index-offset` flag instead?

Reply:
Good idea, the code and description have been updated.